data/korath/korath outfits.txt

```diff
@@ -778,7 +778,7 @@ outfit "Nelmeb GP Hybrid Steering"
 		"frame rate" 6.5
 	"steering flare sound" "ion large"
 	description "Prompted by the grim will to trudge onwards and innovate despite being forced to abandon jump drive technology, Efreti engineers have recently begun to utilize their theoretical knowledge to incorporate gravitational propulsion into their more traditional fusion-based plasma engines."
-	description "	More stable than one would expect from Korath engines this large, these steering thrusters were developed by the Efreti as a means of guiding their most massive ships. Their high energy usage is tempered by the fact that they are usually powered by the most massive Korath fusion reactors."
+	description "	More stable than one would expect from Korath engines this large, these steering thrusters were developed by the Efreti as a means of guiding their most massive ships. Their high energy usage is tempered by the fact that they are usually powered by the most massive Korath Fusion Reactors."
```

Comment:
```suggestion
	description "	More stable than one would expect from Korath engines this large, these steering thrusters were developed by the Efreti as a means of guiding their most massive ships. Their high energy usage is tempered by the fact that they are usually powered by the most massive Korath fusion reactors."
```
"Korath Fusion Reactors" are not an outfit, nor are there any "Fusion Reactor" outfits that the Korath have.